ddp-workspace/projects/ddp-sdk/src/lib/directives/sticky-scroll.directive.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { StickyScrollDirective } from './sticky-scroll.directive';
+import { TestBed } from '@angular/core/testing';
+import { Component, DebugElement } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { WindowRef } from 'ddp-sdk';
+
+const notRecognizableTopSpace = 73;
+const containerHeight = 50;
+// less then container height in order to see the scroll
+const windowHeight = containerHeight - 10 + notRecognizableTopSpace;
```

Comment:
It would be nice to have the 'magic number' as a constant

Reply:
if you mean 10 - well I added a comment which explains that I just need some height less than the container :)
do you prefer me to add a separate const for that? like `const someSmallSpace = 10`?

Reply:
right.
`const someSmallSpace = 10` is more helpful because you can re-use it if needed.

Reply:
done! hope it's better now